backend/tests/lib.rs

```diff
@@ -640,6 +640,24 @@ fn test_user() {
     );
     h4.abort();
     std::thread::sleep(std::time::Duration::from_secs(1));
+
+    // 3. test user_logout
+    // user log out
+    let response = client
+        .get("/users/logout")
+        .remote("127.0.0.1:8000".parse().unwrap())
+        .dispatch();
+    assert_eq!(response.status(), Status::Ok);
+    assert_eq!(response.into_string().unwrap(), "Success");
+    // create burrow: perform a wrong action (user already logout, need authentication)
+    let response = client
+        .post("/burrows")
+        .json(&json!({
+            "description": format!("Second burrow of {}", name),
+            "title": "Burrow 2"}))
+        .remote("127.0.0.1:8000".parse().unwrap())
+        .dispatch();
+    assert_eq!(response.status(), Status::Unauthorized);
```

Comment:
Also add this same request in the beginning of this test function after the creation of `h4` ( I think copy and paste is all you have to do), so that to inform this won't be success before login.